winterfell/utils/httpResponses.js

```diff
@@ -0,0 +1,33 @@
+function httpResponses(app) {
+    this.app = app;
+};
+
+module.exports = httpResponses;
+
+// Http Codes (wrapper around standard http response status codes)
```

Comment:
Ah ok, I would like to have standardized error message constants so I'll keep that for now